### PR TITLE
executor: slow startup for double read.

### PR DESCRIPTION
**Pull request by winoros**
Slowly increase the batch size, small query will benefit from this.

PTAL @shenli @coocood @hanfei1991

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: executor/distsql.go [hunk 1]
```diff
@@ -824,20 +824,21 @@ func (e *IndexLookUpExecutor) executeTask(task *lookupTableTask, goCtx goctx.Con
...
+			taskSizes = append(taskSizes, remained)
+			break
+		} else {
+			taskSizes = append(taskSizes, e.batchSize)
+			remained -= e.batchSize
 		}
-		taskSizes = append(taskSizes, batchSize)
-		total -= batchSize
+	}
+	sum := 0
```
Comment (coocood) [inline executor/distsql.go]: This is not used.

Comment (winoros) [reply executor/distsql.go]: It's used for debug, removed.

File: executor/distsql.go [hunk 2]
```diff
@@ -817,14 +820,17 @@ func (e *IndexLookUpExecutor) executeTask(task *lookupTableTask, goCtx goctx.Con
...
-		taskSizes = append(taskSizes, batchSize)
-		total -= batchSize
+	for remained := len(handles); remained > 0; e.batchSize *= 2 {
+		if e.batchSize > e.ctx.GetSessionVars().IndexLookupSize {
+			e.batchSize = e.ctx.GetSessionVars().IndexLookupSize
+		}
+		if e.batchSize > remained {
+			taskSizes = append(taskSizes, remained)
+			break
+		} else {
```
Comment (coocood) [inline executor/distsql.go]: `else` can be removed.

**Comment by coocood** [comment]
LGTM

**Pull request merged**